screens/SettingsScreen.tsx

```diff
@@ -1,15 +1,28 @@
 import React from 'react';
-import { Text, View } from 'react-native';
+import { Button } from 'react-native';
+
+import { useDispatch } from 'react-redux';
 
 import { NavigationData } from '../navigation/AppNavigator/SettingsNavigator';
+import { logout } from '../redux/auth';
+
+import { StyledView } from './SettingsScreen.styled';
 
 type Props = NavigationData<'Settings'>;
 
 const SettingsScreen: React.FC<Props> = () => {
+  const dispatch = useDispatch();
+
   return (
-    <View>
-      <Text>Settings</Text>
-    </View>
+    <StyledView>
+      {/* TODO: show user name/email and picture if existing */}
```

Comment:
Please remove this comment and move the TODO to our Project Management Excel :smile: